3-things-you-must-know-about-midnight-hacks

```diff
@@ -0,0 +1,45 @@
+{  
+  "author": "Kara Dees",
+  "title": "3 Things You Must Know About Midnight Hacks",
+  "thumbnail": "http://imgur.com/a/YATmL",
```

Comment:
The current setup tries to get the image from the `/images` folder as explained in the `README.md`. However, I do like the idea of using imgur.com so we can discuss the possibility of changing to that process.